assets/js/autosuggest.js

```diff
@@ -1,466 +1,567 @@
-/* eslint-disable camelcase */
-import jQuery from 'jquery';
-import { epas } from 'window';
+/* eslint-disable camelcase, no-underscore-dangle */
+import { findAncestor, escapeDoubleQuotes, replaceGlobally, debounce } from './utils/helpers';
+import 'element-closest';
+import 'promise-polyfill/src/polyfill';
+import 'whatwg-fetch';
+
+const { epas } = window;
 
 /**
  * Submit the search form
- * @param object $localInput
+ *
+ * @param {Node} input - input element
  */
-function submitSearchForm( $localInput ) {
-	$localInput.closest( 'form' ).submit();
+function submitSearchForm(input) {
+	input.closest('form').submit();
 }
 
```

Comment:
```suggestion
/**
	* Set the expanded aria state on the input
	*
	* @param bool haveOptions - whether or not the autosuggest list contains results
	* @param {Node} input - search input
	*/
function toggleInputAria( haveOptions, input ) {
	input.setAttribute( 'aria-expanded', haveOptions );
}

/**
	* Set the active descendant aria attribute input
	*
	* @param string id - id of the currently selected element
	* @param {Node} input - search input
	*/
function setInputActiveDescendant( id, input ) {
	input.setAttribute( 'aria-activedescendant', id );
}
```
Lets assistive technology when the autosuggest is open or closed and which item in the list is currently selected.

Reply:
adding these in, thanks!